src/Sarif/Writers/SarifLogger.cs

```diff
@@ -462,6 +463,27 @@ public void AnalyzingTarget(IAnalysisContext context)
         {
         }
 
+        public void TargetAnalysisCompleted(IAnalysisContext context)
+        {
+        }
+
+        public void LogMemoryUsage(IAnalysisContext context)
+        {
+            var currentProcess = Process.GetCurrentProcess();
+
+            LogConfigurationNotification(
```

Comment:
>LogConfigurationNotification [](http://example.com/codeflow?start=12&length=28)

The memory usage is more like a tool execution notification to me.

Reply:
fixed.